maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -486,7 +487,14 @@ class AndroidDriver(
         .map { parts: Array<String> -> parts[1] }
 
     override fun setLocation(latitude: Double, longitude: Double) {
```

Comment:
Not sure if it is easy to do, but could we also add an integration test for this?

Reply:
There is already one, `051`